OpenRA.Game/Sound/Sound.cs

```diff
@@ -194,17 +200,19 @@ public void Tick()
 			if (MusicPlaying && music.Complete)
 			{
 				StopMusic();
-				onMusicComplete();
+
+				if (onMusicComplete != null)
+					onMusicComplete();
```

Comment:
This should probably set `onMusicComplete` back to `null` in case the next music doesn't want to set a completion action?

Reply:
I removed the null check. This is a bit of a leftover from bug hunting.